package.json

```diff
@@ -49,5 +49,8 @@
     "pretty-quick": "^3.1.1",
     "rimraf": "^3.0.2",
     "serve": "^12.0.0"
+  },
+  "dependencies": {
+    "node": "^17.0.0"
```

Comment:
why did you add node as a dependency?

Reply:
I updated my node js and I guess the package.json got affected 
I made changes already 
Thank you